### DISCUSSION TITLE
A lot of False Positives

### DISCUSSION THREAD

**Discussion by IzacFrank** [Security-Onion-Solutions/securityonion#6602] [Unsupported Versions] [answered]
Hello all,

I'm new to IDS and to security Onion. I see a lot of false positives on the Alerts dashboard. Is there any easy way to eliminate false positives other than threshold? For example host A-H to Host Z on SQL port is OK.

I also see a lot of NAT detection like below, is that normal? It never happened with previous Suricata.
Thank you.

ET INFO Session Traversal Utilities for NAT (STUN Binding Request)		
ET INFO Session Traversal Utilities for NAT (STUN Binding Response)	 
ET INFO Session Traversal Utilities for NAT (STUN Binding Request On Non-Standard High Port)	 
ET POLICY Executable and linking format (ELF) file download 
ET RPC DCERPC SVCCTL - Remote Service Control Manager Access 
ET POLICY DNS Update From External net	

![Capture](https://user-images.githubusercontent.com/93327361/146403140-7db0aeeb-a523-4013-9dab-1b6a43f2ac3f.JPG)

**Comment by dougburks** [comment]
Have you checked the documentation?
https://docs.securityonion.net/en/2.3/managing-alerts.html

**Comment by tomjones1977** [comment]
False positives are a fact of life when establishing a SIEM.  You will need to tune your rules to your network - there are a number of different ways to do this, including thresholding as you mentioned.  Another would be to disable categories such as ET INFO if they are too verbose for your network.

**Reply by IzacFrank** [reply]
Thank you @tomjones1977, I don't have much info alerts, their severity are high usually.

**Reply by IzacFrank** [reply]
Hello @dougburks,

I read the documentation a few times and tried to add suppress rules to the pillar.usage under  local or default salt folders. No mater where I wrote them, they didn't help.

What I don't understand what should I enter to global.sls or minion.sls . The documentation just says enter there but what?

I'm really confused.  

Kind regards,
Isac

**Comment by stevewillia** [comment]
Open and edit your minion file

/opt/so/saltstack/local/pillar/minions/MACHINEID_standalone.sls

Mine is standalone.  Yours may be different

Goto the idstools section and add the SID after disable:
Like below.  it has to be exact spacing/column or it will faIL

idstools:
  config:
    ruleset: 'ETOPEN'
    oinkcode: ''
    urls:
  sids:
    enabled:
    disabled:
     - 2016149
     - 2022024
     - 2029340

Then have salt update everything

sudo salt '*' state.highstate


Steve

**Reply by stevewillia** [reply]
The saving messed up columns

Here is screenshot
<img width="157" alt="Screen Shot 2022-01-04 at 1 10 28 PM" src="https://user-images.githubusercontent.com/77803759/148112513-551f38b1-b293-497e-8bd6-b92911f087eb.png">

**Reply by IzacFrank** [reply]
Thank you @stevewillia but I don't want to disable them, just suppress them. If I add suppress section under disabled, would it be OK?

Thanks again.

**Comment by dougburks** [comment, marked as answer]
@IzacFrank Here's the documentation on thresholding and suppresion:
https://docs.securityonion.net/en/2.3/managing-alerts.html#threshold

You can also take a look at our recent Youtube video:
https://www.youtube.com/watch?v=GQObGTcFl-4

**Reply by IzacFrank** [reply]
Hello @dougburks 
Thank you for the video, it guided me well. The part we should add the **thresholding** section to those files is missing in the documentation and it gave me hard times. 
By the way, in the video, they forgot the indent after the **suppress** like below. Then Suricata service doesn't start.

![image](https://user-images.githubusercontent.com/93327361/148388086-536a7d40-15cc-46ff-82cb-715b214e9227.png)
![highstate-error](https://user-images.githubusercontent.com/93327361/148388377-4b9c34b1-9e80-4577-afa9-00a7b06f6805.JPG)
![error](https://user-images.githubusercontent.com/93327361/148388410-11d3b342-1ac1-4a25-b958-2b336143fe51.JPG)

But it left me the below GRID exclamation mark which I couldn't get rid of.

![image](https://user-images.githubusercontent.com/93327361/148393875-c66d7a86-d47e-4c3e-a4a5-5f4fc5ffb135.png)


**This is the working config from** /opt/so/saltstack/local/pillar/global.sls
I hope it helps people like me to save time.

![ids2](https://user-images.githubusercontent.com/93327361/148388479-23523883-5d2f-4d49-bdd2-0dc34931f992.JPG)


Thanks again.
Isac

**Reply by dougburks** [reply]
> Thank you for the video, it guided me well. The part we should add the **thresholding** section to those files is missing in the documentation and it gave me hard times.

It looks like there are multiple `suppress` examples at https://docs.securityonion.net/en/2.3/managing-alerts.html#threshold.

> By the way, in the video, they forgot the indent after the **suppress** like below. Then Suricata service doesn't start.

We are updating the video with the correct indentation.

**Comment by stevewillia** [comment]
Like Doug said.  You Need to make that change using threshold.

https://docs.securityonion.net/en/2.3/managing-alerts.html#threshold

It’s not in the minion file I showed you but this one for all nodes

/opt/so/saltstack/local/pillar/global.sls

There is a good example in the docs.
